src/AzureDevOpsPolicyConfigurator/Interfaces/ILogger.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace AzureDevOpsPolicyConfigurator
+{
+    /// <summary>
+    /// ILogger interface.
+    /// </summary>
+    internal interface ILogger
+    {
+        /// <summary>
+        /// Logs an ingormation message.
+        /// </summary>
+        /// <param name="message">message</param>
```

Comment:
The comment for `message` can be removed as `message` won't provide additional help in terms of describing the param.

Reply:
how do you want to remove it, it generates a warning then.

Reply:
How about a description like `message to log` then?

Reply:
if you're pleased by that, sure 👍